opensearch_single_kernel/managers/profiles.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""OpenSearch profiles."""
+from typing import List
+
+from opensearch_single_kernel.common.constants import PerformanceType, StartMode
+from opensearch_single_kernel.core.models import (
+    OpenSearchProfile,
+    PeerClusterApp,
+    ProductionProfile,
+    TestingProfile,
+)
+from opensearch_single_kernel.core.state import ClusterState
+from opensearch_single_kernel.managers.base import BaseManager
+from opensearch_single_kernel.utils.helpers import format_unit_name
+from opensearch_single_kernel.utils.topology import ClusterTopology
+from opensearch_single_kernel.workload.base import BaseWorkload
+
+
+class ProfilesManager(BaseManager):
+    """Manage all profile related operations"""
+
+    def __init__(self, state: ClusterState, workload: BaseWorkload):
+        self.state = state
+        self.workload = workload
```

Comment:
```suggestion
    def __init__(self, state: ClusterState, workload: BaseWorkload):
       super().__init__(state, workload)
```
Initializing with super is more appropriate.